static/app/views/settings/project/server-side-sampling/modals/uniformRateModal.tsx

```diff
@@ -88,7 +88,12 @@ function UniformRateModal({
 
   const modalStore = useLegacyStore(ModalStore);
 
-  const {projectStats: projectStats30d, loading: loading30d} = useProjectStats({
+  const {
+    projectStats: projectStats30d,
+    // TODO(sampling): check how to render this error in the UI
```

Comment:
any ideas here @Jesse-Box @matejminar

Reply:
https://github.com/getsentry/sentry/blob/master/static/app/views/releases/detail/overview/releaseComparisonChart/index.tsx#L918-L926

Reply:
yes I saw that before but was not sure... I will try it out...thanks!